core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -105,6 +105,14 @@
       + "<code>MetricReporter</code> interface allows plugging in classes that will be notified "
       + "of new metric creation. The JmxReporter is always included to register JMX statistics.";
   protected static final String METRICS_REPORTER_CLASSES_DEFAULT = "";
+
+  public static final String METRICS_TAGS_CONFIG = "metrics.tags.map";
+  protected static final String METRICS_TAGS_DOC =
+      "A comma separated list of metrics tag entries of the form <tag_name>:<tag_value>. This can"
```

Comment:
Are we using :-separated key-value somewhere else, otherwise I think =-separated might be more common and intuitive.

Reply:
I have based this format off the Kafka config listener.security.protocol.map